docs/_pages/0110-configuration.md

```diff
@@ -22,24 +22,24 @@ next_url: /docs/use-the-source/
 
```

Comment:
There is IDE-generated auto-formatting here.

Can we please revert unrelated changes?